### PR TITLE
Change occurrences of format() to f-strings

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
Also change one occurrence of `%` to `format()`, as changing to an f-string would result in a very long line.

TODO:

- [ ] Unit tests and/or doctests in docstrings
- [x] Tests pass locally
- [ ] Docstrings and API docs for any new/modified user-facing classes and functions
- [x] Changes documented in docs/release.rst
- [x] Docs build locally
- [x] GitHub Actions CI passes
- [x] Test coverage to 100% (Codecov passes)

### CODE REVIEW

**Comment by dstansby** [comment]
Looks good in theory, but is there a ruff rule (or similar) we can enable to automate making sure this change stays in the future?

**Comment by DimitriPapadopoulos** [comment]
The relevant ruff rules will be automatically added by upgrading ruff.

File: c-blosc
Comment (dstansby) [inline c-blosc:1]: Was it intentional to update the c-blosc module here? It seems unrelated to the PR topic, and might be causing the build issues.

Comment (DimitriPapadopoulos) [reply c-blosc:1]: Not intentional. I'll need to read the git manual to manage to see the changes and undo them.

Comment (DimitriPapadopoulos) [reply c-blosc:1]: It's been fixed.

**Review by dstansby — APPROVED** [review]
